components/LifeAtTorchbox/LifeAtTorchboxHero.tsx

```diff
@@ -0,0 +1,93 @@
+import { ArrowLoopRight } from 'components/ArrowLoopRight';
+import { Image } from 'components/Image';
+import { useRef, useState } from 'react';
+import styles from '../../styles/LifeAtTorchboxHero.module.scss';
+import { ImageTypes, VideoTypes } from '../../types/Base';
+import { VideoPlayPauseButton } from './VideoPlayPauseButton';
+
+type LifeAtTorchboxHeroProps = {
+    image: ImageTypes;
+    video?: VideoTypes;
+    subtitle: string;
+    children: React.ReactNode;
+};
+
+export const LifeAtTorchboxHero = ({
+    image,
+    video,
+    subtitle,
+    children,
+}: LifeAtTorchboxHeroProps) => {
+    const [isVideoPlaying, setVideoPlaying] = useState(true);
+    const videoRef = useRef<HTMLVideoElement>(null);
+
+    const toggleVideoPlaying = () => {
+        if (videoRef.current) {
+            if (isVideoPlaying) {
+                videoRef.current.pause();
+            } else {
+                videoRef.current.play();
+            }
+        }
+        setVideoPlaying(!isVideoPlaying);
+    };
+
+    return (
+        <>
+            <svg>
+                <defs>
+                    <clipPath
+                        id="shardClipPath"
+                        clipPathUnits="objectBoundingBox"
+                    >
+                        <path d="M 0.436 0.033 L 0.917 0.34 L 0.519 0.985 L 0.024 0.67 Z " />
+                        <path d="M 0.996 0.406 L 0.77 0.79 L 0.967 0.908 L 1.207 0.533 L 1.03 0.429 Z" />
+                    </clipPath>
+                </defs>
+            </svg>
+            <div className={styles.heroContainer}>
+                <Image
+                    src={image.url}
+                    alt={image.description}
+                    className={styles.mobileImage}
+                />
+                {video ? (
+                    <div className={styles.videoContainer}>
+                        <video
+                            className={styles.video}
```

Comment:
**Thought:** You can add `cursor: pointer;` here and it behaves pretty nicely with the hitbox if that makes any sense (only changes when hovering over the video and not the cropped out parts), this would highlight it's clickable but it might not be needed though...

Reply:
I've just noticed theres a play/pause button however this is hidden for me on firefox, if its suppose to be hidden I'd suggest sr-only with no svg or styling. otherwise I think it might need some more testing on different size devices.

Reply:
Good spot, I need to double check my styles as it's not rendering correctly, thank you